build_poh.sh

```diff
@@ -29,19 +31,18 @@ do
     fi
 done
 
-set -e
+# get the Open Hospital version
+version=$(grep -m 1 "</version>" pom.xml | grep -o "[0-9.]*")
+
+# clone core, gui and doc repositories
+git clone -b v$version https://github.com/informatici/openhospital-core.git core
+git clone -b v$version https://github.com/informatici/openhospital-gui.git gui
+git clone https://github.com/informatici/openhospital-doc.git doc
```

Comment:
also doc is tagged before each release, so we can clone it in the same way

Reply:
Done